src/osd/OSD.cc

```diff
@@ -2569,6 +2569,18 @@ float OSD::get_osd_recovery_sleep()
     return cct->_conf->osd_recovery_sleep_hdd;
 }
 
+float OSD::get_osd_delete_sleep()
+{
+  float osd_delete_sleep = cct->_conf.get_val<double>("osd_delete_sleep");
```

Comment:
if you use the legacy way (cct->_conf->osd_delete_sleep) for these the options will be injectable at runtime

Reply:
@jdurgin I thought that too, but tried it anyway without the legacy options and it worked.

bin/ceph daemon osd.0 config set osd_delete_sleep 60

2018-10-25 16:50:16.521 ... _delete_some Delete scheduled at 2018-10-25 16:51:16.524557

Reply:
oh maybe that's new since the centralized config in mimic